websites/P/Planet Minecraft/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "283380624687890433",
+		"name": "cardinalsystem"
+	},
+	"service": "Planet Minecraft",
+	"description": {
+		"en": "Planet Minecraft is a family friendly community that shares and respects the creative works and interests of others. They have a variety of entertaining ways to explore and upload content, socialize and find like minded creative members from around the world that love the game of Minecraft."
+	},
+	"url": "www.planetminecraft.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/YcaaiHk.png",
```

Comment:
The [logo](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px.